src/libraries/Microsoft.Extensions.Caching.Memory/tests/MemoryCacheSetAndRemoveTests.cs

```diff
@@ -406,6 +406,29 @@ public void ClearClears()
             }
         }
 
+        [Fact]
+        public void SetNullCallback_NotAllowed_ArgumentException()
+        {
+            var cache = CreateCache();
+            const string someKey = "test";
+            var entry = cache.CreateEntry(someKey);
+
+            var options = new MemoryCacheEntryOptions();
+
+            var notNullCallback = new PostEvictionCallbackRegistration()
+            {
+                EvictionCallback = (_, _ , _, _) => Console.WriteLine()
```

Comment:
```suggestion
                EvictionCallback = (_, _ , _, _) => {}
```

That way people looking at the code won't think we are writing to the Console in tests.